student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(age, id, classroom, name: 'Unknown', parent_permission: true)
+    super(age, id, name, parent_permission)
```

Comment:
- [ ] As already pointed above, only age, name, and parent_permission are required to create a Person(Student).